staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -327,22 +322,41 @@ var (
 			},
 		},
 	)
-	SuggestedFlowSchemaWorkloadLeaderElection = newFlowSchema(
-		"workload-leader-election", "leader-election", 200,
+	// We add an explicit rule for endpoints-controller with high precedence
```

Comment:
@liggitt @deads2k - PTAL

I'm personally not a huge fan of this solution, but the below seem to be much less controversial and is already moving us in the right direction. So for the sake of making progress let's do something and we can get back to the previous discussion once I'm back from vacation.

Reply:
I think this approach will allow separating the controllers needing high throughput from most of those using it for leader election.